chain-adapter/src/main/kotlin/ChainAdapter.kt

```diff
@@ -0,0 +1,50 @@
+package chainadapter
+
+import com.github.kittinunf.result.map
+import com.rabbitmq.client.ConnectionFactory
+import config.RMQConfig
+import jp.co.soramitsu.iroha.java.IrohaAPI
+import model.IrohaCredential
+import mu.KLogging
+import sidechain.iroha.util.ModelUtil
+
+class ChainAdapter(private val rmqConfig: RMQConfig) {
+    private val logger = KLogging().logger
+
+    fun run() {
```

Comment:
`close()` should be added as well

Reply:
In this case shutting down the process will be close :)